module/Photo/view/photo/album/index-new.phtml

```diff
@@ -0,0 +1,265 @@
+                        data-raw-url="<?= $this->url('photo/photo_download', ['photo_id' => $item->getId()]); ?>"
+                        data-vote-url="<?= $this->url('photo/photo/vote', ['photo_id' => $item->getId()]); ?>"
+                    >
+                        <img
+                            class="lazy-load"
+                            data-width="<?= $smallSize['w'] ?>" data-height="<?= $smallSize['h'] ?>"
+                            data-src="<?= $this->glideUrl()->getUrl($item->getPath(), $smallSize) ?>"
+                            alt=""
+                            data-size="<?= $smallSize['w'] . 'x' . $smallSize['h'] ?>"
+                        >
+                    </a>
+                    <figcaption style="display:none">
+                        <?= $this->partial('partial/metadata.phtml', ['photo' => $item]) ?>
+                    </figcaption>
+                    <figtags style="display:none">
+                        <?= $this->partial('partial/tags-new.phtml', ['photo' => $item]) ?>
+                    </figtags>
+                </figure>
+            <?php endforeach; ?>
+        </div>
+    </div>
+    <!-- Root element of PhotoSwipe. Must have class pswp. -->
+    <div class="pswp" tabindex="-1" role="dialog" aria-hidden="true">
+
+        <!-- Background of PhotoSwipe.
+             It's a separate element as animating opacity is faster than rgba(). -->
+        <div class="pswp__bg"></div>
+
+        <!-- Slides wrapper with overflow:hidden. -->
+        <div class="pswp__scroll-wrap">
+
+            <!-- Container that holds slides.
+                PhotoSwipe keeps only 3 of them in the DOM to save memory.
+                Don't modify these 3 pswp__item elements, data is added later on. -->
+            <div class="pswp__container">
+                <div class="pswp__item"></div>
+                <div class="pswp__item"></div>
+                <div class="pswp__item"></div>
+            </div>
+
+            <!-- Default (PhotoSwipeUI_Default) interface on top of sliding area. Can be changed. -->
+            <div class="pswp__ui pswp__ui--hidden">
+
+                <div class="pswp__top-bar">
+
+                    <div class="pswp__counter"></div>
+
+                    <button class="pswp__button_glyph pswp__button--close glyphicon-remove" title="Close (Esc)"></button>
+                    <button class="pswp__button pswp__button--fs" title="Toggle fullscreen"></button>
+                    <button class="pswp__button pswp__button--zoom" title="Zoom in/out"></button>
+
+                    <button class="pswp__button_glyph pswp__button--download glyphicon-download-alt" title="<?= $this->translate('Download') ?>" data-toggle="tooltip" data-placement="bottom"></button>
+                    <button class="pswp__button_glyph pswp__button--like glyphicon glyphicon-thumbs-up" title="<?= $this->translate('Vote for photo of the week') ?>" data-toggle="tooltip" data-placement="bottom"></button>
```

Comment:
Is it possible to show the user if he has already voted on this photo? (using the color highlight I suggested in another comment)

Reply:
Yes, however persisting this after a page refresh is slightly more complicated (due to limited options to preload additional variables in to the state of the viewer). So that will be added in a future version.